### PR TITLE
Problem: dragonberry is not updated

### PR DESCRIPTION
**Pull request by yihuang**
👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [ ] Have you read the [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md)?
- [ ] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [ ] Have you rebased your work on top of the latest master? 
- [ ] Have you checked your code compiles? (`make`)
- [ ] Have you included tests for any non-trivial functionality?
- [ ] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-org-chain/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
-## UNRELEASED
+*Oct 15, 2022*
+
+## v0.8.2
 
 ### Bug Fixes
 
 - [#693](https://github.com/crypto-org-chain/cronos/pull/693) Clear `From` field when building cosmos tx in `eth_sendTransaction`.
 - [cronos#715](https://github.com/crypto-org-chain/cronos/pull/715) Add option `iavl-disable-fastnode` to disable iavl fastnode indexing migration, and enable `iavl-cache-size` to have effect on app (backport #714).
+- [#]() Dragonberry update.
```
Comment (yihuang) [inline CHANGELOG.md:11]:
```suggestion
- [#739](https://github.com/crypto-org-chain/cronos/pull/739) Dragonberry update.
```

**Review by tomtau — APPROVED** [review]

File: go.mod [hunk 2]
```diff
@@ -156,7 +156,8 @@ require (
 replace (
 	// TODO: fix keyring upstream
 	github.com/99designs/keyring => github.com/crypto-org-chain/keyring v1.1.6-fixes
-	github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.45.9-0.20220920132636-63a7c45caa76
+	github.com/confio/ics23/go => github.com/cosmos/cosmos-sdk/ics23 v0.45.9
```
Comment (julienrbrt) [inline go.mod:159]:
```suggestion
	github.com/confio/ics23/go => github.com/cosmos/cosmos-sdk/ics23/go v0.8.0
```

Comment (mmsqe) [reply go.mod:159]: We point to ics23 [correct](https://github.com/cosmos/cosmos-sdk/tree/v0.45.9/ics23) pythically, no `cosmos-sdk/ics23/go` different from replace methioned in the [link](https://forum.cosmos.network/t/ibc-security-advisory-dragonberry/7702/16).

**Review by mmsqe — COMMENTED** [review]
I rmb we need fixed memo length with [rand](https://github.com/mmsqe/cosmos-sdk/blob/dbe1dba5a6dfa118412a6a3ea795c5f1b746eee7/simapp/helpers/test_helpers.go#L26) seed for sim test.

**Pull request merged**